server/src/main/java/org/elasticsearch/action/search/SearchResponse.java

```diff
@@ -557,6 +573,82 @@ public String toString() {
         }
     }
 
+    /**
+     * Extension of the Clusters class for CCS uses. It adds the number of remote clusters
+     * included in a search and whether ccs_minimize_roundtrips is true.
+     *
+     * Like Clusters, this is an immutable class.
+     *
+     * Unlike Clusters, (for now at least) the new fields are not Writeable. They are only
+     * used during CCS searches for accounting on the primary coordinator.
```

Comment:
it may be worth mentioning async search here as in fact this info is only useful as part of async search in the coordinating node?